### PR TITLE
[#561] Add variable to handle margins in base/typography

### LINKED ISSUE #561: Add variable to handle margins in base/typography
Default of no margin is good, but it’d be great to be able to override that

### PR DESCRIPTION
**Pull request by planktonic**
Fixes #561 

The following changes are contained in this pull request:
- Default margins for heading elements and paragraphs can now be specified with the `$margin-heading` and `$margin-paragraph` variables respectively
- The default margin for paragraphs is now `0`

Delete if not applicable:

- [x] Your changes have been added to the `unreleased` section of [CHANGELOG.md](../CHANGELOG.md)

### CODE REVIEW

File: scss/bitstyles/generic/_normalize.scss [hunk 1]
```diff
@@ -170,14 +170,6 @@ button:-moz-focusring,
   outline: 1px dotted ButtonText;
 }
 
-/**
- * Correct the padding in Firefox.
- */
-
-fieldset {
-  padding: size.get('m');
```
Comment (maltoe) [inline scss/bitstyles/generic/_normalize.scss:178]: drive-by?

**Review by maltoe — APPROVED** [review]
LGTM 👍

File: scss/bitstyles/generic/_normalize.scss (hunk 1 above)
Comment (planktonic) [reply scss/bitstyles/generic/_normalize.scss:178]: Yes kinda :) It’s related to the issue, in that it’s about default spacing applied to elements at this early stage, which is why I found it. This property gets re-specified later in the forms file (using a variable that can be overridden in projects, which for typography was the purpose of the PR), so this line was not needed
